lib/index.js

```diff
@@ -237,18 +237,11 @@ module.exports = {
     return new MergeTrees([
       concatBundle(emberFiles, {
         outputFile: 'ember.js',
-        footer: "require('ember');",
+        footer: "require('@ember/-internals/bootstrap');",
       }),
 
       concatBundle(emberTestingFiles, {
         outputFile: 'ember-testing.js',
-        footer: `
-          var testing = require('ember-testing');
-          Ember.Test = testing.Test;
-          Ember.Test.Adapter = testing.Adapter;
-          Ember.Test.QUnitAdapter = testing.QUnitAdapter;
-          Ember.setupForTesting = testing.setupForTesting;
```

Comment:
Where do these go?

Reply:
Updated to make them lazy getters on the global Ember object, but we still need to `require('ember-testing');` because it reopens some classes eagerly and tests fail without it.